packages/react-components/src/AccessibilityScenarios/Slider.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Slider } from '@fluentui/react-components/unstable';
```

Comment:
The same for Slider too, IMO it should be imported from `@fluentui/react-slider`, same as Label it is on line 11.

Reply:
Yes, thanks.